tensorflow/training/docker/2.1.1/py3/cu101/Dockerfile.gpu

```diff
@@ -156,7 +156,7 @@ RUN mkdir -p /root/.ssh/ \
 WORKDIR /
 
 RUN ${PIP} --no-cache-dir install --upgrade \
-    pip \
+    pip==20.2.1 \
```

Comment:
Why is this necessary?

Reply:
Using pip newer than 20.2.1 leads to a failure of horovod deployment